ds-ux-guidelines/src/pages/test-page/test-page.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import Layout from "../../ds-components/layout/layout"
+import "./test-page.scss"
+// import ContentHeader from "../../ds-components/content-header/contentHeader"
+// import SEO from "../../ds-components/seo"
+
+/*
+  sm: 1-5 <599
+  m: 1-9 600-1023
+  lg: 1-13 1024-1439
+  xl: 1-13 <1440
+*/
+
+const TestPage = () => (
+  
+
+    <div className="test cbp-grid">
+      <div className="cbp-col-1-2  cbp-col-md-2-4 cbp-col-lg-3-6">
+        Indented Row.
+      </div>
+      <div className="cbp-col-full">
+        Full Row.
+      </div>
+
+      <div class="cbp-col-2-12">
+        Indented two columns.
+      </div>
```

Comment:
Another example of utilizing the grid - going from medium to small breakpoint has an awkward phase of min-width
```
    <div className="cbp-grid" style={{ paddingTop: "2em" }}>
      <div
        className="cbp-col-12 cbp-col-md-2 cbp-col-lg-3"
        style={{ border: "1px solid black" }}
      >
        <figure>
          <img
            style={{ width: "100%" }}
            src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcSD9bDQFRm7xWzg0G1YHqgosDTCIbRzjSm8TtOw3cVUldzTHByX&usqp=CAU"
          ></img>
          <figcaption>Cat Pic</figcaption>
        </figure>
      </div>
      <div
        className="cbp-col-12 cbp-col-md-2-5 cbp-col-lg-3-6"
        style={{ border: "1px solid black" }}
      >
        <p>{/* Lorem Ipsum Here  */}</p>
        <br />
        <p>{/* Lorem Ipsum Here  */}</p>
        <br />
        <p>{/* Lorem Ipsum Here  */}</p>
      </div>
      <div
        className="cbp-col-12 cbp-col-md-7-1 cbp-col-lg-9-3"
        style={{ border: "1px solid black" }}
      >
        {/* Lorem Ipsum Here  */}
      </div>
    </div>
```